### PR TITLE
Feature api client connect

### PR DESCRIPTION
**Pull request by zachwreed**
**This PR includes:**
- Update to Review psql table
- Insert statements into Review psql table
- UI implementation of games table, game page, user review, and reviews table
- UI implementation of review forms
- Adds notifications when inserting, deleting, or updating reviews
- Adds notifications when an error is present in games, game, or review state
- Redux states for game, games, and reviews
- Connects games and reviews API requests to Redux store

**Items to resolve:**
- Route handling for all games, game, and addGame page
- React Bootstrap Table onClick handling
- Connect AddGame component to Redux store

### CODE REVIEW

File: api/routes/games.js [hunk 1]
```diff
@@ -211,9 +211,14 @@ router.post('/:game_id/reviews', (req, res) => {
       val = parseInt(req.body[field]) || null;
     }
 
+        console.log(field, val)
+
+
     query.values.push(val);
   }
 
+  console.log("game id", req.params.game_id)
```
Comment (Jedeu) [inline api/routes/games.js]: Can we delete the console.logs?

Comment (zachwreed) [reply api/routes/games.js]: removed

**Review by Jedeu — APPROVED** [review]
Looks good to me. Not too familiar with the Redux stuff so a demo during the meeting will be awesome.

**Review by mobilemelody — APPROVED** [review]
Looks good to me

**Review by Jedeu — APPROVED** [review]

**Pull request merged**